packages/api/cms-api/generate-schema.ts

```diff
@@ -76,11 +78,12 @@ async function generateSchema(): Promise<void> {
     const FileDependentsResolver = DependentsResolverFactory.create(File);
 
     const schema = await gqlSchemaFactory.create([
+        AzureAITranslatorModule,
```

Comment:
No need to import the module here

Reply:
529f1fa699f335d4898d5f0a2571434f23d9123f